style_editor.html

```diff
@@ -9689,7 +9728,7 @@
   Run();
 }
 
-function SetToAndFormat(str) {
+function SetToAndFormat(str, event) {
```

Comment:
Is this event thing related to the color sort somehow, or did it sneak in with other changes?

Reply:
I have to try to remember all this from a while ago and relearn what 's going on. I think event is te listening to the menu checkbox state?

Reply:
Turns out this has sneaked in from a future PR for enabling and disabling appropriate tabs.